src/main/java/sorald/Main.java

```diff
@@ -71,6 +76,20 @@ public JSAP defineArgs() throws JSAPException{
 		opt.setHelp("Max number of fixes per rule. Default: Integer.MAX_VALUE (or all)");
 		jsap.registerParameter(opt);
 
+		opt = new FlaggedOption(Constants.ARG_MAX_FILES_PER_SEGMENT);
+		opt.setLongFlag(Constants.ARG_MAX_FILES_PER_SEGMENT);
+		opt.setStringParser(JSAP.INTEGER_PARSER);
+		opt.setDefault("" + 6500);
+		opt.setHelp("Max number of files per loaded segment for segmented repair. It should be >= 3000 files per segment. Default: 6500 (256mb Jvm) . ");
+		jsap.registerParameter(opt);
+
+		opt = new FlaggedOption(Constants.ARG_REPAIR_STRATEGY);
+		opt.setLongFlag(Constants.ARG_REPAIR_STRATEGY);
+		opt.setStringParser(JSAP.STRING_PARSER);
+		opt.setDefault(RepairStrategy.DEFAULT.name());
+		opt.setHelp("Type of repair strategy. DEFAULT - load everything without splitting up the folder in segments, SEGMENT - splitting the folder into smaller segments and repair one segment at a time (need to specify --maxFilesPerSegment if not default). (default: DEFAULT)");
+		jsap.registerParameter(opt);
```

Comment:
I also already told you that the arg `repairStrategy` should appear before `maxFilesPerSegment` since first one decides the strategy for the repair, and if the segmented one is chosen, one sets the max files per segment.